components/template/Templates.vue

```diff
@@ -39,9 +39,9 @@
           v-else
           class="no-templates d-flex flex-column align-items-center justify-content-center w-100"
         >
-          <nuxt-img
+          <img
```

Comment:
why no nuxt image?

Reply:
@ankitsinghaniyaz I want to test if removing nuxt-img working fine in production.